### PR TITLE
autocorrect: init at 2.6.2

### PR DESCRIPTION
**Pull request by ghost**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/text/autocorrect/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{ lib, rustPlatform, fetchFromGitHub, Security }:
```
Comment (JulienMalka) [inline pkgs/tools/text/autocorrect/default.nix:1]:
```suggestion
{ lib, stdenv, rustPlatform, fetchFromGitHub, Security }:
```

**Review by JulienMalka — CHANGES_REQUESTED** [review]

File: pkgs/tools/text/autocorrect/default.nix (hunk 1 above)
Comment (candyc1oud) [reply pkgs/tools/text/autocorrect/default.nix]: I fixed it, thanks

**Comment by candyc1oud** [comment]
@JulienMalka I fixed it 
thanks

**Comment by JulienMalka** [comment]
@candyc1oud why are you closing this ?

**Comment by candyc1oud** [comment]
> @candyc1oud why are you closing this ?

@JulienMalka Sorry, you haven't replied to me for so long that I thought you didn't care about this PR

**Comment by JulienMalka** [comment]
> @JulienMalka Sorry, you haven't replied to me for so long that I thought you didn't care about this PR

@candyc1oud it is not for me to care or not care about this PR, we are a whole team of people that give some of their time to review all PRs on Nixpkgs (there is a lot of them) and to try to bring all of them in a mergable state if possible. Sometimes this process is long we often are only doing this on our free time. You have to be patient. If you feel that your PR is good to be merged, you can ping a reviewer or you can try to post it on the discourse, there is a subject "PR ready for review".

**Pull request merged**